data/credits/en.credits

```diff
@@ -1,6 +1,44 @@
 # Lines longer than 78 characters will be wrapped automatically
 # The following line is for reference
 ##############################################################################
+(<color_cyan>0.I</color>):</color>
+<color_white>Original author:</color>    <color_white>Project Manager:</color>
+<color_yellow>Whales</color> (<color_dark_gray>retired</color>)    <color_yellow>KevinGranade</color>
+
+<color_white>Current Main Developers:</color>
+<color_yellow>RenechCDDA</color> - Improvements to NPC factions, basecamps, and vehicles.
+
+<color_white>Special thanks to:</color>
+<color_yellow>Standing-Storm</color> - Creator and maintainer of the enormous 'Mind over Matter' mod.
+<color_yellow>Procyonae</color> - Highways!!!
+<color_yellow>PatrikLundell</color> - An absurd amount of bugfixes and important safety changes to mapgen, tripoint infrastructure, and other critical systems.
+<color_yellow>Anton Simakov 'GuardianDll'</color> - CDDA's newest merger, somehow they've also found the time to learn C++ and still be a big-time contributor.
+<color_yellow>RenechCDDA</color> - Improvements to NPC factions, basecamps, and vehicles.
+<color_yellow>Maleclypse</color> - Xedra Evolved maintainer and over 2k merges this cycle
+<color_yellow>Holli</color> - For their dedicated work on guns, guns, and more guns!
+<color_yellow>b3brodie</color> - Tons of infrastructure support for spells, bettering both mods and the base game.
+<color_yellow>Milopetilo</color> - Awesome work on new mapgen, paramaters for existing mapgen, item group overhauls, and more.
+<color_yellow>ShnitzelX2</color> - Electronic storage for items and cleaning up 10 years of technical debt by moving items to be loaded with generic_factory. (Don't be jealous, but they're my favorite.)
+<color_yellow>alef</color> - Keeping the game building so we can keep developing and playing.
+<color_yellow>John Candlebury</color> - Aftershock. Did you know you can own and fly a colossal spaceship in Aftershock now? Yeah.
+<color_yellow>Daniel Brooks 'db48x'</color> - For fundamental work on our beautiful new ImGui windows, the UI we've always wanted.
+<color_yellow>moxian</color> - Mission-critical work on build and IWYU infrastructure. They've made every developer's life easier.
+<color_yellow>ehughsbaird</color>, <color_yellow>Karol1223</color>, <color_yellow>Andrew Krieger 'akrieger'</color>, <color_yellow>mqrause</color>, and <color_yellow>SariusSkelrets</color>.
```

Comment:
```suggestion
<color_yellow>Karol1223</color> - For all things critter, creature, and crawling. 
<color_yellow>ehughsbaird</color>, <color_yellow>Andrew Krieger 'akrieger'</color>, <color_yellow>mqrause</color>, and <color_yellow>SariusSkelrets</color>.
```
Karol is like me with guns but with critters instead

Reply:
Not sure about this one, I looked at their post-0.H work and it wasn't much.